modules/test-structure/test_structure.go

```diff
@@ -72,6 +72,16 @@ func CopyTerraformFolderToTemp(t testing.TestingT, rootFolder string, terraformM
 		return filepath.Join(rootFolder, terraformModuleFolder)
 	}
 
+	fullTerraformModuleFolder := filepath.Join(rootFolder, terraformModuleFolder)
+
+	exists, err := files.FileExistsE(fullTerraformModuleFolder)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !exists {
+		t.Fatal(os.ErrNotExist, fullTerraformModuleFolder)
```

Comment:
Will this show a clear error message?

Reply:
This will result in: 

`TestTerraformAwsExamplePlan: test_structure.go:82: file does not exist ../no/examples/terraform-aws-example`

Reply:
That's decent, but would be slightly better if it said "directory" instead of file. If you extract + reuse the error type as mentioned above, that'll do the trick.